daemon/api.go

```diff
@@ -283,8 +284,8 @@ func getPackagesInfo(c *Command, r *http.Request) Response {
 
 	for _, v := range bags {
 		m := v.Map(nil)
-		name := m["name"]
-		origin := m["origin"]
+		name, _ := m["name"].(string)
```

Comment:
because if the map holds something that isn't a string, I want the same behaviour as if the map didn't hold the key at all: use the default/null value.